rslib/src/notes/mod.rs

```diff
@@ -350,12 +350,11 @@ impl Collection {
         self.set_current_notetype_id(note.notetype_id)
     }
 
-    #[cfg(test)]
-    pub(crate) fn update_note(&mut self, note: &mut Note) -> Result<OpOutput<()>> {
+    pub fn update_note(&mut self, note: &mut Note) -> Result<OpOutput<()>> {
         self.transact(Op::UpdateNote, |col| col.update_note_inner(note))
     }
 
-    pub(crate) fn update_notes_maybe_undoable(
+    pub fn update_notes_maybe_undoable(
```

Comment:
Might be better to leave this one private unless there was a particular use-case you had in mind. The backend only exports this because some legacy Python/add-on code depends on the old system of deferring transactions, but that should not be an issue with projects using the Rust API directly.

Reply:
To get the other changes into the next rc, I'll revert this line without waiting for feedback, but feedback is still welcome.